libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -308,6 +311,51 @@ bool IRGeneratorForStatements::visit(FunctionCall const& _functionCall)
 			")\n";
 		break;
 	}
+    case FunctionType::Kind::Assert:
+    case FunctionType::Kind::Require:
+    {
+        solAssert(arguments.size() > 0, "Expected at least one parameter for require/assert");
+
+        for (auto& arg: arguments)
+            arg->accept(*this);
+
+        if (arguments.size() == 1)
+        {
+            m_code << Whiskers("if iszero(<cond>) { <function> }")
+                ("cond", m_context.variable(*arguments[0]))
+                (
+                    "function",
+                    FunctionType::Kind::Assert == functionType->kind() ?
+                        "invalid()" : "revert(0, 0)"
+                ).render();
+            break;
+        }
+
+        auto const errHash =
+            (u256(FixedHash<4>::Arith(FixedHash<4>(dev::keccak256("Error(string)")))) << (256 - 32));
+
+        ABIFunctions abiFunctions(m_context.evmVersion(), m_context.functionCollector());
```

Comment:
Don't we have that in the utils?

Reply:
Unfortunately not, no. Not sure it's worth having either, as it can easily be created on demand

Reply:
Ok!